Python/Product/Debugger/PtvsdVersionHelper.cs

```diff
@@ -71,8 +76,12 @@ internal class PtvsdDebuggerVersion {
     }
 
     internal class PythonVersionInfo {
+        /// <summary>
```

Comment:
Normally you can omit any field that you are not interested in, and json deserialization will just skip them (as a bonus, it ends up being faster). If you plan on using them later, then feel free to keep them.

Reply:
We actually plan on using this data to show process and interpreter info in 'Attach to Process' UI.